democracylab/emails.py

```diff
@@ -423,6 +424,7 @@ def send_email(email_msg, email_acct=None):
         email_msg.subject = test_email_subject
         email_msg.body = test_email_body
         email_msg.to = [settings.ADMIN_EMAIL]
+        email_msg.cc = ["thisIsADummyEmailAddress",]
```

Comment:
You can just blank this out.  The important thing is to record this information above in test_email_body